packages/styled-components/components/image/themes/default.js

```diff
@@ -0,0 +1,69 @@
+import styled, { css } from 'styled-components';
+
+/**
+ * Get some CSS specifically when we're displaying the image with a specific
+ * image ratio.
+ *
+ * @param {object} props Component props.
+ * @return {string} CSS for aspect ratio.
+ */
+const getAspectRatioStyles = (props) => {
+  const { aspectRatio } = props;
+
+  if (aspectRatio) {
+    return css`
+      height: 100%;
+      left: 0;
+      position: absolute;
+      top: 0;
+      width: 100%;
+    `;
+  }
+
+  return '';
+};
+
+/* eslint-disable import/prefer-default-export */
```

Comment:
IIRC you shouldn't need to disable the default export rule if you're exporting multiple variables in a single file.